src/app/app-modules/reset-password/reset-password.component.html

```
@@ -1,104 +1,100 @@
-<div class="support-wrapper">
-  <div class="support-card">
-    <header class="support-header">
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)